Made Get RESTPlus and used Black to make code pep8 compliant

sushanthsamala opened the pull request:
- Added GET call that is RESTPlus compliant
- Used Black to make code pep8 compliant

sushanthsamala committed:
Made Get RESTPlus and used Black to make code pep8 compliant

sushanthsamala committed:
Commented out behave tests from travis

sushanthsamala committed:
cleanup

sushanthsamala committed:
cleanup

Mu-xsan commented:
Hi, Sushanth! Why did you comment out behave tests? Don't we need to automate BDD tests every time we try to merge?

sushanthsamala commented:
> Hi, Sushanth! Why did you comment out behave tests? Don't we need to automate BDD tests every time we try to merge?

@Mu-xsan Um how will we behave test if we have no endpoints? I think we can uncomment it once we reach a point where we have enough endpoints. Currently there is only one endpoint that I added with this PR, I deleted all others no buttons will be working for a while. I'm assuming that we are going with the approach that we will be deleting the old endpoints and build the new ones from scratch as it will only take 1 sprint right?

sushanthsamala committed:
Added back endpoints

sushanthsamala committed:
added old endpoints

sushanthsamala committed:
cleanup

sushanthsamala committed:
cleanup

sushanthsamala committed:
cleanup

sushanthsamala committed:
cleanup

sushanthsamala committed:
cleanup

sushanthsamala committed:
cleanup
